packages/config/config/devices/0x010f/fgd212_3.5.json

```diff
@@ -171,41 +171,41 @@
 					"value": 0
 				},
 				{
-					"label": "Dimmer operates on auto-calibration settings",
+					"label": "Dimmer operates on auto-calibration",
 					"value": 1
 				}
 			]
 		},
 		"15": {
-			"label": "Burnt out bulb detection",
-			"description": "Percentage value of power variation",
+			"label": "Burnt out bulb detection power variation (LOAD ERROR)",
+			"description": "Percentage value of standard power consumption as measured during calibration, 0 = detection disabled",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 99,
 			"defaultValue": 30
 		},
 		"16": {
-			"label": "Time delay of a burnt out bulb",
-			"description": "Time of delay for power variation detection",
+			"label": "Burnt out bulb detection time delay (LOAD ERROR)",
+			"description": "0 = detection disabled",
 			"valueSize": 2,
+			"unit": "seconds",
 			"minValue": 0,
 			"maxValue": 255,
 			"defaultValue": 5
 		},
 		"19": {
 			"label": "Forced switch on brightness level",
-			"description": "Dimmer 2 will always set this brightness level.",
+			"description": "Percentage value, 0 = disabled, previous value is used.",
```

Comment:
`0` should be an `"option"`

Reply:
`0` moved to `option`. Would it be correct to use the `"unit" : "%"` here? I guess not? So we leave that in the description?